crates/storage/db/src/abstraction/table.rs

```diff
@@ -101,19 +101,17 @@ pub trait TableImporter<'tx>: for<'a> DbTxMut<'a> {
         source_tx: &R,
         from: Option<<T as Table>::Key>,
         to: <T as Table>::Key,
-    ) -> Result<(), Error> {
+    ) -> Result<(), Error>
+    where
+        T::Key: Default,
+    {
         let mut destination_cursor = self.cursor_write::<T>()?;
         let mut source_cursor = source_tx.cursor_read::<T>()?;
 
-        for row in source_cursor.walk(from)? {
+        let from = from.unwrap_or_default();
+        for row in source_cursor.walk_range(from..to)? {
```

Comment:
judging from previous code, this should be changed
```suggestion
        for row in source_cursor.walk_range(from..=to)? {
```